practices/graded-browser-support.md

```diff
@@ -76,21 +89,21 @@ We couple the loading of JavaScript to the loading of the enhanced CSS. We use t
 
 #### Internet Explorer 10 support
 
-As of January 2017, we consider Internet Explorer 10 as Grade-C. This is mainly due to its lack of support for certain modern features plus a very low usage. IE10 also stopped receiving security updates in January 2016 from its manufacturer.
+As of January 2017, we consider Internet Explorer 10 as a Core browser. This is mainly due to its lack of support for certain modern features plus very low usage numbers. IE10 also stopped receiving security updates in January 2016 from its manufacturer.
```

Comment:
`we consider Internet Explorer 10 as a Core browser` -> `we consider Internet Explorer 10 a Core browser` ?